cypress/integration/04-main-table.spec.ts

```diff
@@ -465,6 +465,9 @@ describe('Open saved dataset in MainTable', () => {
     mainTablePage.mainTable.exportReport.click()
     mainTablePage.mainTable.exportExcel.click()
     cy.wait('@reportDownload')
+    cy.readFile('./cypress/downloads/Dataset_from_autotests.xlsx').should(
```

Comment:
I'd check the file content.

Reply:
Have some troubles here because of the encoding

Reply:
Maybe transfer from xlsx to csv

Reply:
Could be resolved later, but it's better to check file content.